verilog/parser/verilog.y

```diff
@@ -4844,11 +4937,15 @@ gate_instance_or_register_variable
     { $$ = std::move($1); }
   /* TODO(fangism): arrays should not be declared with port connections */
   /* TODO(b/36706412): support anonymous instances */
```

Comment:
@hzeller does this PR address this bug?

Reply:
There is now a different error message when using the test file in bug #268 
Instead of the semicolon at the end, it complains about the open parenthesis:

```
$ bazel-bin/verilog/tools/syntax/verible-verilog-syntax --show_diagnostic_context /tmp/b.sv 
/tmp/b.sv:19:15: syntax error at token "("
USER_DEF_PRIM (UDP_OUT_X, UDP_IN_X, vpwr, vgnd) ;
              ^
```